### PR TITLE
Focus breadcrumb on route change

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Focus current active breadcrumb on route change and announce contents of current folder to screenreader

### CODE REVIEW

**Review by kulmann — CHANGES_REQUESTED** [review]
Found one issue that I didn't think about before. This now steals the focus away from the nav directly after clicking in the left sidebar. As far as I understand this behaviour is only supposed to happen upon navigating within the file list or through the breadcrumbs.

**Comment by pascalwengerter** [comment]
> Found one issue that I didn't think about before. This now steals the focus away from the nav directly after clicking in the left sidebar. As far as I understand this behaviour is only supposed to happen upon navigating within the file list or through the breadcrumbs.

Tried preventing it with what we discussed, checking if we're staying within the same route and only setting the focus on the last breadcrumb item if that's the case

File: packages/web-app-files/src/mixins/accessibleBreadcrumb.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+      const announcement = this.$gettextInterpolate(
+        'containing %{filesCount} files and %{folderCount} folders',
+        { filesCount: this.activeFilesCount.files, folderCount: this.activeFilesCount.folders }
+      )
+
+      const translatedHint =
+        this.activeFilesCount.folders > 0 || this.activeFilesCount.files > 0
+          ? announcement
+          : this.$gettext('This folder has no content')
```
Comment (kulmann) [inline packages/web-app-files/src/mixins/accessibleBreadcrumb.js:23]:
```suggestion
          : this.$gettext('This folder has no content.')
```

File: packages/web-app-files/src/mixins/accessibleBreadcrumb.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  methods: {
+    accessibleBreadcrumb_focusAndAnnounceBreadcrumb(sameRoute) {
+      const activeBreadcrumb = last(
+        document.getElementById('files-breadcrumb').children[0].children
+      )
+      const activeBreadcrumbItem = activeBreadcrumb.lastChild
+
+      const announcement = this.$gettextInterpolate(
+        'containing %{filesCount} files and %{folderCount} folders',
```
Comment (kulmann) [inline packages/web-app-files/src/mixins/accessibleBreadcrumb.js:16]: Don't make the distinction between files and folders. Just use `items` and with correct pluralization. `This folder contains %{ itemCount } item.` for singular form and `This folder contains %{ itemCount } items.` plural form.

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
LGTM 🚀

**Pull request merged**